pkg/status/status.go

```diff
@@ -339,7 +339,7 @@ func (s *status) monitorClusterStatus() {
 		case <-s.stopCh:
 			// If the stopCh is closed, the operator will exit and CO should
 			// be set to failing.
```

Comment:
This should say
`// be set to degrading`

Reply:
@awgreene Nice catch! Updated in latest.